uktrade/data-hub-frontend
# Refactor - Move referrals into companies

## Description of change

The referrals should sit beneath companies as a sub app. As I am moving this all routes need to be updated and breadcrumbs updated to reflect the correct hierarchy.

## Test instructions
Find a company in Staging that has referrals. The following urls should render with breadcrumbs:

1. Referral details - /companies/{companyId}/referrals/{referralId}

2. Referral help - /companies/{companyId}/referrals/{referralId}/help

3. Send referral - /companies/{companyId}/send-referral

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
